src/view/range.js

```diff
@@ -349,3 +408,29 @@ export default class Range {
 		return this.createFromPositionAndShift( Position.createBefore( item ), 1 );
 	}
 }
+
+// Function used by getEnlagred and getShrinked methods.
+function enlagreShrinkStartSkip( value ) {
```

Comment:
Those two functions differ only in one string. Maybe we can do them as function generator function (like converters).